README.md

```diff
@@ -23,19 +23,23 @@ Lifecycle Version | Platform APIs              | Buildpack APIs |
 
 ### Build
 
-* `detector` - chooses buildpacks (via `/bin/detect`)
-* `analyzer` - restores launch layer metadata from the previous build
-* `restorer` - restores cache
-* `builder` -  executes buildpacks (via `/bin/build`)
-* `exporter` - creates image and stores cache
+Either:
+* `detector` - Chooses buildpacks (via `/bin/detect`) and produces a build plan.
+* `analyzer` - Restores layer metadata from the previous image and from the cache.
```

Comment:
Is it `and` or `or`?
Can it happen that there will be both previous image and a cache and the `analyzer` will restore metadata from both of them?
Can we have conflicts in such case?

Reply:
It restores metadata for launch layers from the previous image and metadata for cached layers from the cache.

https://github.com/buildpacks/spec/blob/main/platform.md#layer-analysis

Reply:
Thanks!